### DISCUSSION TITLE
Black disc in counter

### DISCUSSION THREAD

**Discussion by Richy0007** [jomjol/AI-on-the-edge-device#2132] [General] [answered]
### The Feature

The water meter from Zenner (MNK-P-N) has a rotating black disc with numbers in addition to the rotating pointers. Unfortunately, the black disc is not recognized.

https://www.ikz.de/fileadmin/_processed_/1/c/csm_HAUT_20220513_018_01_07_picture-0001_d1ef42cc92.jpg

https://zenner.de/products/hwz_mnk-p/

It would be nice if you could learn it in the software

**Comment by jomjol** [comment]
The problem with this is, that is has totally different logic (no pointer, but numbers). This would be a major change and might need a third type of counter + major changes in the software algorithm. 
Currently a lot of work.

**Comment by Slider0007** [comment, marked as answer]
@Richy0007: I have the same model. Just use 0,1 and 0,01 with enabled extended resolution. The works without any issue and accepable resolution. No issues with the not perfect visible and a little disorted part of x0,01... 

![image](https://user-images.githubusercontent.com/115730895/222258190-e88fbb3c-2a43-40b8-9336-bce0bfd06684.png)
